targets/XPlatCppSdk/source/test/TestApp/PlayFabEventTest.cpp

```diff
@@ -91,32 +91,35 @@ namespace PlayFabUnit
         auto pfEvent = std::dynamic_pointer_cast<const PlayFab::PlayFabEvent>(event);
         auto pfResponse = std::dynamic_pointer_cast<const PlayFab::PlayFabEmitEventResponse>(response);
 
-        // Handle successful event delivery.
-        if (PlayFab::PlayFabErrorCode::PlayFabErrorSuccess == pfResponse->playFabError->ErrorCode)
-        {
-            ++eventPassCount;
-            eventFailLog += pfEvent->GetName() + " was sent successfully " +
-                "in the batch #" + std::to_string(pfResponse->batchNumber) + " "
-                "of " + std::to_string(pfResponse->batch->size()) + " events. "
-                "HTTP code: " + std::to_string(pfResponse->playFabError->HttpCode) +
-                ", app error code: " + std::to_string(pfResponse->playFabError->ErrorCode) + "\n";
-
-            // Keep track of the highest batch number.
-            eventBatchMax = (pfResponse->batchNumber > eventBatchMax) ? pfResponse->batchNumber : eventBatchMax;
-        }
-        // Handle failed event delivery.
-        else
-        {
-            ++eventFailCount;
-            eventFailLog += pfEvent->GetName() + " received an error back " +
-                "in the batch #" + std::to_string(pfResponse->batchNumber) + " "
-                "of " + std::to_string(pfResponse->batch->size()) + " events. "
-                "HTTP code: " + std::to_string(pfResponse->playFabError->HttpCode) +
-                ", app error code: " + std::to_string(pfResponse->playFabError->ErrorCode) +
-                ", HTTP status: " + pfResponse->playFabError->HttpStatus +
-                ", Message: " + pfResponse->playFabError->ErrorMessage +
-                "\n";
-        }
+		if (pfResponse->playFabError != nullptr)
```

Comment:
Please change your Visual Studio/Text Editor to use 4 spaces instead of tabs.
And then replace all tab characters in this file with 4 spaces.